lona/app.py

```diff
@@ -462,6 +461,5 @@ def run(
         # start server
         run_server(
             args=server_args,
-            app=self.aiohttp_app,
```

Comment:
Why is this line removed?

Reply:
The ``app`` is now created inside the ``LonaServer`` (see https://github.com/lona-web-org/lona/pull/224/files#diff-71b8e0eb0eada6eab6aa80a7d9faa08f0a543d3b9c8fc4e1c425f34b9d6f7b51R100 ). This way we have access to the settings and know the value of ``AIOHTTP_CLIENT_MAX_SIZE``.

This argument to ``command_line/run_server.py::run_server()`` has also been removed (see https://github.com/lona-web-org/lona/pull/224/files#diff-5096a6e3173b4dcd96c074d6a72b4cbf0c76af7b6581ef36144abe35dc06c4fdR22 ).
To still have access to the aiohttp-server app I just grab it from the internal state of the ``LonaServer`` (see here https://github.com/lona-web-org/lona/pull/224/files#diff-5096a6e3173b4dcd96c074d6a72b4cbf0c76af7b6581ef36144abe35dc06c4fdR35 ). This is not really elegant but is in line with the code around it.